lib/install/bin/webpack-dev-server.tt

```diff
@@ -23,20 +26,39 @@ end
 begin
   dev_server = YAML.load_file(CONFIG_FILE)["development"]["dev_server"]
 
-  DEV_SERVER_HOST = "http#{"s" if args('--https') || dev_server["https"]}://#{args('--host') || dev_server["host"]}:#{args('--port') || dev_server["port"]}"
+  public_host = args('--host') || dev_server["host"]
+  port        = args('--port') || dev_server["port"]
+  https       = ARGV.include?('--https') || dev_server["https"]
 
 rescue Errno::ENOENT, NoMethodError
   puts "Webpack dev_server configuration not found in #{CONFIG_FILE}."
   puts "Please run bundle exec rails webpacker:install to install webpacker"
   exit!
 end
 
+begin
+  server = TCPServer.new(LISTEN_IP_ADDR, port)
```

Comment:
@ytbryan `TCPSocket` is client socket for making a connection to a server and listen for a response whereas `TCPServer` is for running a server and binding it to a port and IP. We are testing if our new server can bind to the given port and IP.